docs/OperatorBestPractices.md

```diff
@@ -1,26 +1,59 @@
-# Operator Best Practices
+# Backing Service Best Practices
```

Comment:
I would rename the file itself as well - instead of `OperatorBestPractices.md` to `BackingServicesBestPractices.md`.

Reply:
Good point!

Reply:
Done!

Reply:
need to update readme too..